plugins/com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml

```diff
@@ -12,4 +13,14 @@
          requiredSourceLevel="1.5">
      </compilationParticipant>
   </extension>
+  
+  <extension 
+      id="blacklistedClass" 
+      name="App Engine Whitelist Marker"
+      point="org.eclipse.core.resources.problemmarkers">
```

Comment:
The bundle symbolic name (BSN) is the human-readable name for a bundle.  In your case, `com.google.cloud.tools.eclipse.appengine.whitelist`.

`FrameworkUtil` is a helper class from OSGi that will find the bundle that contains a class.  The Eclipse Extension Registry tracks the extensions contributed by a bundle; the extension registry is supposed to be independent of OSGi, so it uses terms like contributors (= bundle) that have names ('namespaces' = BSN).